client/src/index.js

```diff
@@ -4,4 +4,16 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+import {
+  BrowserRouter as Router,
+  Route,
+  Link,
+  NavLink
+} from "react-router-dom";
```

Comment:
You only need to import what you use. So for index.js, you only need BrowserRouter as Router. Everything else you can get rid of.